src/Test/contlinear.jl

```diff
@@ -1472,8 +1472,6 @@ function partial_start_test(model::MOI.ModelLike, config::TestConfig)
     MOI.empty!(model)
     @test MOI.is_empty(model)
 
-    @test MOI.supports(model, MOI.VariablePrimalStart(), MOI.VariableIndex)
```

Comment:
Hmm? This shouldn't be removed.

Reply:
This allow solvers not supporting variable primal start to pass the test with a warning